content/docs/cve/cveTeams.md

```diff
@@ -1,55 +1,57 @@
  *  **Team Identifier:**  - Used by the applicants to login and join the exam; also used in permalinks for the session and the team on HamStudy.
  *  **Team Name:** - Displays along with the team identifier in the session listing on HamStudy.
- *  **E-mail Address:** - Should contain a valid email address for the administrator of the team.
- *  **Team Website:** - Enter the url (i.e. http://…) for the website for the club or group associated with this team. Leave blank if you do not have one.
- *  **My team permissions:** - The personal permissions settings for the team you are viewing. (not editable and set by the owner/co-owner of the session)
+ *  **Contact E-mail Addresses:** - Should contain a valid email addresses for the administrators of the account.
+ *  **Team Website:** - Enter the url (i.e. http://…) for the website for the club or group associated with this team account. Leave blank if you do not have one.
+ *  **My team permissions:** - The personal permissions settings for the team account you are viewing. (not editable and set by the owner/co-owner of the session)
  *  **Delegated Privileges:** (In order to list sessions for a given VEC one or more VEs who are at least co-owners must delegate their team lead permissions to the team. Note that sessions must still have a valid team lead assigned from the team, but anyone with Scheduling permissions may schedule sessions for any VEC for which the team has delegated permissions.) - Only those with session manager permissions in ExamTools will see VECs in the Delegate Privileges list.  Select the VEC for which you would like the team schedulers to be able to create sessions for.  If selected, the VEC will display below the feature.
  
- ## Add, Edit, and Delete Members
+ ## Add, Edit, and Delete Administers
  
- *  **Members:** - A list of those VEs who are part of your team management.  This list does not include everyone of your VEs, but rather those few who assist with the scheduling, coordinating and running of the sessions.
- *  Click on the **+** (plus sign) next to **Members:** to add members. 
- *  Use the **Edit** (pencil and paper icon) or **Delete** (trash can icon) to manage members on the list.
- *  Click **Edit**, to **Change team member permissions**.
- *  The team member's *call sign and license class* will display along with the **Name:** which may use the Display Name from the VE's Profile, if entered.
- *  **Credentials:** (view which VECs they are accredited with) - A little blue winner's cup will display, if they are a team lead/session manager in ET for that VEC.
+ *  **Adminstrators:** - A list of those VEs who are part of your team management.  This list does not include everyone of your VEs, but rather those few who assist with the scheduling, coordinating and running of the sessions.
+ *  Click on the **+ INVITE** next to **Administrator:** to add administrators whoc will need to accept your invitation. 
```

Comment:
```suggestion
 *  Click on the **+ INVITE** next to **Administrator:** to add administrators who will need to accept your invitation. 
```
Per your "tired" comment in Discord, maybe don't request a review until you've taken a second look and fixed typos.